src/WebJobs.Extensions.DurableTask.Analyzers/Analyzers/ActivityFunction/NameAnalyzer.cs

```diff
@@ -34,7 +34,7 @@ private static string GetClosestString(string name, IEnumerable<string> availabl
         {
             foreach (var activityInvocation in calledFunctions)
             {
-                if (!availableFunctions.Select(x => x.FunctionName).Contains(activityInvocation.Name))
+                if (!availableFunctions.Select(x => GetCleanedFunctionName(x.FunctionName)).Contains(GetCleanedFunctionName(activityInvocation.Name)))
```

Comment:
We already store FunctionName accounting for the nameof() case. This method would only benefit our activityInvocation name.